packages.go

```diff
@@ -0,0 +1,68 @@
+//
```

Comment:
Please move all the additional packages module to a new pull request. It is not related to the Windows PR or the NIC Configuration or memory banks work.